### PR TITLE
refactor: Refactor get vulnerabilities command to use client instead of kubectl

### PR DESCRIPTION
**Pull request by hypnoglow**
Fixes https://github.com/aquasecurity/starboard/issues/142

### CODE REVIEW

**Comment by hypnoglow** [comment]
Tested it locally, the main difference after this patch is that resource list is now CRD-typed:

Old (master):

```yaml
apiVersion: v1
items:
- apiVersion: aquasecurity.github.io/v1alpha1
  kind: VulnerabilityReport
  # redacted...
kind: List
metadata:
  resourceVersion: ""
  selfLink: ""
```

New (with this patch):

```yaml
apiVersion: aquasecurity.github.io/v1alpha1
items:
- apiVersion: aquasecurity.github.io/v1alpha1
  kind: VulnerabilityReport
  # redacted...
kind: VulnerabilityReportList
metadata:
  resourceVersion: "2666"
  selfLink: /apis/aquasecurity.github.io/v1alpha1/namespaces/default/vulnerabilityreports
```

File: pkg/cmd/root.go [hunk 1]
```diff
@@ -71,3 +71,7 @@ func initFlags() {
 		}
 	})
 }
+
+func init() {
+	_ = starboardv1alpha1.AddToScheme(GetScheme())
```
Comment (hypnoglow) [inline pkg/cmd/root.go:76]: This was required to make printer aware of custom CRD schemes.

**Comment by hypnoglow** [comment]
I haven't found any tests for `get vulns` command, am I missing something? If not, I'm not sure if I should add one in this PR, or it should be a separate change.

**Comment by danielpacak** [comment]
> I haven't found any tests for `get vulns` command, am I missing something? If not, I'm not sure if I should add one in this PR, or it should be a separate change.

We don't have any integration test for get vulns command yet. And we can add it in a separate pull request.

**Comment by danielpacak** [comment]
Great contribution @hypnoglow ! Could you please rebase the PR branch so I can proceed with approval and merge to master.

**Comment by hypnoglow** [comment]
Rebased

**Review by danielpacak — APPROVED** [review]
LGTM!

**Pull request merged**